modules/oe_content_entity/modules/oe_content_entity_organisation/config/install/core.base_field_override.oe_organisation.oe_stakeholder.status.yml

```diff
@@ -0,0 +1,21 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - oe_content_entity_organisation.oe_organisation_type.oe_stakeholder
+id: oe_organisation.oe_stakeholder.status
+field_name: status
+entity_type: oe_organisation
+bundle: oe_stakeholder
+label: Published
+description: ''
+required: false
+translatable: false
+default_value:
+  -
+    value: 1
+default_value_callback: ''
+settings:
+  on_label: 'On'
+  off_label: 'Off'
+field_type: boolean
```

Comment:
We don't need to export base field overrides.

Reply:
done